pkg/gui/presentation/stash_entries.go

```diff
@@ -25,6 +25,7 @@ func getStashEntryDisplayStrings(s *models.StashEntry, diffed bool) []string {
 	if icons.IsIconEnabled() {
 		res = append(res, textStyle.Sprint(icons.IconForStash(s)))
 	}
-	res = append(res, textStyle.Sprint(s.Name))
+
+	res = append(res, textStyle.Sprint(s.Line()))
```

Comment:
We should do exactly what we do for branches here, and make the recency a separate column (the first one, it should go before the icon). Also, color it cyan [like we do for branches](https://github.com/jesseduffield/lazygit/blob/66b608b2f9857296645f964b67b821e431d7b807/pkg/gui/presentation/branches.go#L101). (No need for a recencyColor variable of course.)